dds/FACE/FaceTSS.h

```diff
@@ -398,6 +398,8 @@ void register_callback(FACE::CONNECTION_ID_TYPE connection_id,
   } else {
     DDS::DataReaderListener_var listener = new Listener<Msg>(callback, connection_id);
     readers[connection_id]->dr->set_listener(listener, DDS::DATA_AVAILABLE_STATUS);
+    // TODO: Do we need to invoke the listener?
+    // TODO: Can calbacks be invoked concurrently?
```

Comment:
Do we have answers for these questions?

Reply:
@mitza-oci Does FACE say we need to invoke the listener here?  If so, do we need to prevent a callback from being invoked concurrently?

Reply:
I couldn't find any specific requirement for this.  My suggestion would be to leave the code in this function as it is.